week-2-practise/level-1/6-map.js

```diff
@@ -11,7 +11,7 @@ function tidyUpString(str) {
 }
 
 var mentors = ["/Daniel ", "irina ", " Gordon", "ashleigh "];
-var mentorsTidy; // ONLY EDIT THIS LINE
+var mentorsTidy = mentors.map(tidyUpString); // ONLY EDIT THIS LINE
```

Comment:
Unfortunately .map() is a bit more complicated to use - you can't just pass it a function. Have a look here for the syntax https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map#Mapping_an_array_of_numbers_using_a_function_containing_an_argument

Mapping is like looping over an array, meaning that you perform the same action on each individual item in an array. In order to do that you need to 
1) choose a name to reference each item (you can choose anything, example:`item` or `mentor` or `x`),
2) tell the .map what name you've chosen to call each one of its items (here we've chosen the name 'item')
```
mentors.map(function(item) {
 //
}
```
3) inside the map(), say what you want to do to each item - and don't forget the parameters! Here, `tidyUpString` expects a parameter, which will be each item in your mentors array. 
```
mentors.map(function(item) {
   return tidyUpString(item));
}
```
This way, `tidyUpString` will be performed on each element in the mentors array, one after the other. The result will be a new array, saved under the variable `mentorsTidy`.

We will go over this in class on Sunday :)